### ISSUE TITLE
Can't spawn in signs on 1.13.2 (and possibly more items)

### ISSUE THREAD

**Issue by haloflooder** [EssentialsX/Essentials#2790]
### Information

**Full output of `/ess version`**:

```
Server version: 1.13.2-R0.1-SNAPSHOT git-Paper-645 (MC: 1.13.2)
EssentialsX version: 2.17.1.15
LuckPerms version: 4.4.1
PlaceholderAPI version: 2.10.3
Vault version: 1.7.2-b107
Citizens version: 2.0.25-SNAPSHOT (build 1746)
EssentialsXChat version: 2.17.1.15
```

**Server log**: 
There's no stacktrace but here you go.
```
haloflooder issued server command: /i dirt
haloflooder issued server command: /i grass
haloflooder issued server command: /i sign
haloflooder issued server command: /i glass
```

**EssentialsX config**: https://gist.github.com/haloflooder/5f96ba75c8d88dad1a15cebddd56d458

### Details

**Description**  
You can't give players/yourself a sign. Possibly other items relating to the 1.14 update but haven't checked yet.

**Steps to reproduce**  
Run 1.13.2 server
Join server
Run `/i sign` command
???
Get `Error: null` in chat!

**Expected behavior**  
I expected to get a sign. What a ripoff. 5/7 would use again

**Screenshots**  
![Just for fun](https://i.imgur.com/gsf9EY8.png)

### Temporary Fix
I temporarily fixed the issue by changing the sign portion of `items.json` from
```json
  "sign": "oak_sign"
```
to
```json
  "sign": {
    "material": "SIGN"
  }
```
or I could've just grab a sign from creative but that's like. more work man.

**Comment by x0Z3ro0x** [comment]
Confirmed, and there are a few more items as well that do this. Highly annoying. Your fix doesn't work for 1.14 here is the fix for 1.14.

Change item.json line number 9294 - 9297 to the following -

```
    "sign": {
        "material": "OAK_SIGN"
    },
	"oaksign": "oaksign",
```

I will fix any other items I find as well.

**Comment by pop4959** [comment]
Confirmed as well. Probably an issue resulting from signs having been renamed internally in 1.14.

@x0Z3ro0x If you can provide a list of other items you've found that do this as well, it may help the Essentials team.

**Comment by x0Z3ro0x** [comment]
I would be happy to. I am going through now and testing the items. I will have a full list for you by tonight.

**Comment by triagonal** [comment]
Possibly related to #2732 - results in same error (as the OP) when attempting to spawn cactus green on 1.13.2

**Comment by x0Z3ro0x** [comment]
Alright, I know I am a bit late but that is because I wanted to be extremely thorough. I have went through your entire items.json twice now. From what I can tell, 1.14 changes the names behind the different wood type items. Signs, and Doors where the ones I found. Boats, Trap Doors, and pressure plates all works. Same with planks.

All other blocks function correctly.

EDIT:

> Possibly related to #2732 - results in same error when attempting to spawn cactus green on 1.13.2

This is not present in 1.14. I just tested. Using /i cactus indeed provides a green cactus.

**Comment by mdcfe** [comment]
This appears to be a problem arising from the merged `items.json` which comes with EssentialsX 2.17.0 and above. (For those interested, it consists of entries from both `genItemsFlat` (deprecated, 1.13.x) and `ItemDbGenerator` (WIP, 1.13.x-1.14.x).)

The `items.json` from `ItemDbGenerator` for both 1.13.x and 1.14.x can be downloaded from [here](https://ci.ender.zone/job/EssentialsX-ItemDbGenerator/lastSuccessfulBuild/artifact/target/items.json) - note that it's missing many entries and isn't ready for production, but the entries present should be consistent and work on both 1.13 and 1.14.

**Comment by pop4959** [comment]
@md678685 Just updating this thread to say that it seems #2897 doesn't fix this yet. I feel like that's still a bug because in 1.13+ "sign" technically isn't in the game but it would make sense to give an oak sign as it would in versions prior to 1.13.

Expected behaviour:
Before 1.13: /i sign gives minecraft:sign
1.13 and later: /i sign gives minecraft:oak_sign

On the other hand, at the very least /i oak_sign works in the PR, whereas it seems not to on current versions.

**Comment by mdcfe** [comment]
This is fixed by d92f5ce71c96e1f0a7fb93eeab28ed97eb13e7a0 - you can download the latest build from the [downloads page](https://essentialsx.cf/downloads).
